manifests/c/CeruleanStudios,LLC/Trillian/6.4.0.5/CeruleanStudios,LLC.Trillian.installer.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate
+
+PackageIdentifier: CeruleanStudios,LLC.Trillian

```

Comment:
Duplicate of Trillian.Trillian.

Reply:
PR for updating existing manifest has been created: https://github.com/microsoft/winget-pkgs/pull/15988
Sorry for the confusion.